modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/HeaderDropdown.tsx

```diff
@@ -24,14 +24,17 @@ const HeaderDropdown: FC<IHeaderDropdown> = ({
 	deleteElement,
 }) => {
 	const flags = useFeatureFlag();
-
 	const [active, setActive] = useState<boolean>(false);
 	const [
 		{
 			isViewOnly,
 			objectLayout: {objectLayoutTabs},
 		},
 	] = useContext(LayoutContext);
+	const handleOnClick = (handler: Function) => {
```

Comment:
@cgmonte @MariMalta Please, add a blank line here. The `setActive` function is being defined in this same block as its usage.